/*
 * Copyright (C) 2026 tonikelope
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 */
package com.tonikelope.coronapoker.table;

/** Service-provider boundary implemented by optional table renderer modules. */
public interface TableRendererProvider {

    String id();

    TableRenderer create(TableCommandSink commandSink);
}
